Packs/HatchingTriage/Integrations/HatchingTriage/HatchingTriage.yml

```diff
@@ -12,19 +12,28 @@ configuration:
   type: 0
 - display: API Key
   name: API Key
-  required: true
+  required: false
   type: 4
   additionalinfo: The API Key to use for the connection.
+  displaypassword: API Key
+  hiddenusername: true
```

Comment:
```suggestion
```
Please remove.

Reply:
Hey Moshe! Can you explain this further? Why do these specific lines need to be removed?

Reply:
Hey @NicCas
Sure, Because of the `hidden: true` (of line 20) so the whole parameter is already hidden.